app/client/e2e/login.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { test, expect } from '@playwright/test';
+
+test('login page should have email form', async ({ page }) => {
+    await page.goto('/login');
+    await expect(page.locator('[data-test-id=login-email-text]')).toHaveText('Email *Email *');
```

Comment:
Could you explain why the toHaveText is formatted like this and se
![Screenshot from 2022-05-02 16-20-39](https://user-images.githubusercontent.com/14946745/166341595-2d1c033c-bedb-466f-856c-167b1b6195c0.png)
ems to have an invisible Unicode character?

Reply:
I was actually a bit confused about that as well. I initially did `tohaveText('Email')` because that was how it was set in our code; However, after running the test, playwright said it expected to see `'Email *Email *'` as the label on the email input field so I temporarily put that in my testing code to see if it even passes. I was in the progress of looking into other methods I could potentially use to go around this issue. (same thing for password field)